src/common/responses.py

```diff
@@ -99,32 +100,36 @@ async def wrapper_decorator(*args, **kwargs) -> TResponse | JSONResponse:
             except ValidationException as e:
                 if logger.level <= logging.DEBUG:
                     traceback.print_exc()
-                logger.error(e)
+                logger.debug(e)
                 return JSONResponse(e.dict(), status_code=status.HTTP_400_BAD_REQUEST)
             except NotFoundException as e:
                 if logger.level <= logging.DEBUG:
                     traceback.print_exc()
-                logger.error(e)
+                logger.debug(e)
                 return JSONResponse(e.dict(), status_code=status.HTTP_404_NOT_FOUND)
             except BadRequestException as e:
                 if logger.level <= logging.DEBUG:
                     traceback.print_exc()
-                logger.error(e, extra={"Traceback": get_traceback()})
-                logger.error(e.dict())
+                logger.debug(e, extra={"Traceback": get_traceback()})
```

Comment:
Is the error stack now printed twice? Both on line 112 and 113? Or is extra= not visible in console?